### ISSUE TITLE
Quick how to resubmit messages 

### ISSUE THREAD

**Issue by jayjlawrence** [OpenSMTPD/OpenSMTPD#877]
I don't know if this is the best way but it got the job done for me.

I had some bad routing that caused entries to be stuck in the queue and needed to be resubmitted or "requeued"

Thanks to a discussion from 2014 I got enough info to make this one liner. Obviously take off the bash at the end and try a few lines by hand.

```bash
smtpctl show queue|awk -F\| '{print "smtpctl show message " $1 " | sendmail -f " $5 " -- " $6 ";smtpctl remove "$1 }' | bash
```

Perhaps you can do it in two steps too

```bash
smtpctl show queue|awk -F\| '{print "smtpctl show message " $1 " | sendmail -f " $5 " -- " $6 }'  > step1
```

```bash
smtpctl show queue|awk -F\| '{print "smtpctl remove "$1 }' > step2
```

and then back together as one
```bash
chmod a+x step[12]; ./step1; ./step2
```

As passer by - I am done with my email server work for 2018 and hopefully 2019 - please accept this info for what it is worth.

**Comment by greecemunky** [comment]
Thanks @jayjlawrence - this saved my bacon!

My OpenSMTPD 6.0.3p1 instance (used solely as an internal relay) had messages stuck in queue due to error `mta event=error reason=IO Error: error:141A318A:SSL routines:tls_process_ske_dhe:dh key too small` which I believe coincided with an OpenSSL update (I performed a Debian dist-upgrade recently).  I assume my current smarthost uses an undesirable DH key length.  All entries in queue had status `Network error on destination MXs`.

I changed the smarthost specified in /etc/smtpd.conf (to another internal OpenSMTPD server using the same smarthost however its OpenSSL is yet to be updated) and restarted OpenSMTPD however the mail messages were still trying the former server/s (derived from the single smarthost entry).  I also tried `smtpctl schedule all` after they failed again however it made no difference.

After scratching my head trying to get them to requeue and ignore the former server, I recalled seeing this post.  As a precaution I ran the step1/step2 commands separately and voila!

There were only 37 messages (some of which appear to have been NDRs regarding other messages stuck in queue!), but looking at the queue I can sense some are important so it's great to get them out.
